package.json

```diff
@@ -43,17 +44,16 @@
     "rollup-stream": "^1.14.0",
     "run-sequence": "^1.2.2",
     "sass-lint": "1.10.0",
+    "standard": "^8.6.0",
     "supertest": "^2.0.1",
     "through2": "^2.0.1",
     "uglify-save-license": "^0.4.1",
     "vinyl-buffer": "^1.0.0",
-    "vinyl-source-stream": "^1.1.0",
-    "change-case": "^3.0.0"
-
+    "vinyl-source-stream": "^1.1.0"
   },
   "scripts": {
     "start": "gulp start",
-    "test": "gulp test"
+    "test": "standard && gulp test"
```

Comment:
Wouldn't bother changing here but I like to do:

```
"lint": "standard",
"test": "npm run lint && gulp test"
```

Which allows me to easily tack on additional linters/coverage/type checkers later.